analyze/analyze.rs

```diff
@@ -1119,6 +1121,18 @@ impl Ord for DiffEntry {
     }
 }
 
+impl serde::Serialize for DiffEntry {
```

Comment:
Any reason to implement this by hand rather than to use `#[derive(Serialize)]`?

Reply:
Ah! This was implemented manually because the `delta` value should have its sign shown even if it is positive. The derived implementation matched tests for the negative values, but positive values were missing the `+`.

If there's a way to do this with serde macros though, I'm happy to follow up and fix that :)